### PR TITLE
docs: separate docs tokens from components tokens

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/5022228012

### CODE REVIEW

File: src/components/IconButton/__stories__/iconButton.stories.module.scss [hunk 1]
```diff
@@ -31,23 +31,23 @@
...
     height: 49px;
     width: 100%;
-    border-bottom: 1px solid var(--layout-border-color);
+    border-bottom: 1px solid var(--sb-layout-border-color);
   }
   &Content {
     height: 200px;
     width: 100%;
-    background-color: var(--grey-background-color);
+    background-color: var(--sb-primary-background-hover-color);
```
Comment (YossiSaadi) [inline src/components/IconButton/__stories__/iconButton.stories.module.scss:51]: changed from a regular color variant to hover color variant
intentional?

File: src/components/Loader/__stories__/Loader.stories.mdx [hunk 2]
```diff
@@ -149,7 +149,7 @@ If there is a need for color customization, css `color` attribute of a parent co
 
 <Canvas>
   <Story name="Custom colors">
-    <div style={{ color: "var(--color-dark-red)" }}>
+    <div style={{ color: "#bb3354" }}>
```
Comment (YossiSaadi) [inline src/components/Loader/__stories__/Loader.stories.mdx:152]: intentionally hard coded?

File: src/components/AlertBanner/__stories__/alertBanner.stories.scss [hunk 3]
```diff
@@ -18,14 +18,6 @@
...
-    .monday-alert-banner__close-button-wrapper {
-      background: var(--private-color);
-    }
-    .monday-alert-banner__inner {
-      background: var(--private-color);
-    }
-    .monday-alert-banner__alert-banner-close-btn {
-      background: var(--private-color);
-    }
+    background: var(--sb-color-sofia_pink);
```
Comment (YossiSaadi) [inline src/components/AlertBanner/__stories__/alertBanner.stories.scss:21]: there were three different classes before, is this covers them all?
just making sure this is intentional and not a testing leftover or a tech debt 😄

File: src/components/Loader/__stories__/Loader.stories.mdx (hunk 2 above)
Comment (SergeyRoyt) [reply src/components/Loader/__stories__/Loader.stories.mdx:152]: Wasn't sure what to do here, cause we want to get rid of any usages of monday-ui-style tokens in the stories files, but I didn't want to add this color to the vibe-storybook-components just for this use case
WDYT?

File: src/components/IconButton/__stories__/iconButton.stories.module.scss (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/IconButton/__stories__/iconButton.stories.module.scss:51]: Changed it cause I didn't want to add new color token to the vibe-storybook-components just for this

File: src/components/Loader/__stories__/Loader.stories.mdx (hunk 2 above)
Comment (YossiSaadi) [reply src/components/Loader/__stories__/Loader.stories.mdx:152]: Let's create a tech debt ticket for it

**Review by YossiSaadi — APPROVED** [review]
Great work 👏🏻

**Pull request merged**